platform/mac/do_mac_bundle.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Script to generate mac application bundles for KOReader
+# shellcheck disable=SC2164
+
+if [ -z "${1}" ]; then
+    echo "${0}: can't find KOReader build, please specify a path"
+    exit 1
+else
+    INSTALL_DIR="${1}"
+    VERSION="$(cut -f2 -dv "${1}/koreader/git-rev" | cut -f1,2 -d-)"
+fi
+
+{
+    echo '<?xml version="1.0" encoding="UTF-8"?>'
```

Comment:
Wouldn't that be more legible with a heredoc?

Reply:
done